staticDHCPd/databases/oracle.sql

```diff
@@ -1,7 +1,6 @@
 CREATE DATABASE dhcp;
 ALTER SESSION SET CURRENT_SCHEMA = dhcp;
 
-Sergey Ko, [21.01.20 16:22]
```

Comment:
Oh, whoops, I saw this one before, but forgot to flag it while I was checking the Oracle docs to learn about variable type differences from Postgres.